api/mod/tests/conftest.py

```diff
@@ -5,20 +5,31 @@
 from pathlib import Path
 from typing import Generator
 
+import alembic
 import pytest
+from alembic.config import Config
 from fastapi.testclient import TestClient
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
 
 from mod.src.app import APP
+from mod.src.database.database import get_db
+from mod.src.models.user import UserCreate
 from mod.src.settings import SETTINGS
 
+temp_folder = Path(tempfile.mkdtemp(prefix="annoto"))
+SETTINGS.database_url = f"sqlite:///{temp_folder.joinpath('test_db.db')}"
+
+engine = create_engine(SETTINGS.database_url, connect_args={"check_same_thread": False})
+
```

Comment:
Also: Will this database even be empty again for every single test execution? o.O

Reply:
Ok, I see, the database will be wiped by rolling back the transaction. That's probably quicker than what I thought of :thinking:

Reply:
The problem with the datafolder is that it is regenerated (in the session fixture) for every test, currently. So that would also require to rebuild the whole database for each test :/

Reply:
damn
I'd suggest to leave it as it is, also because I have the feeling that we will no longer need to rebuild our data folder for each and every test as soon as annotations are stored in our db as well - and then we could just remove the test fixture that does so and keep this one, right?